### PR TITLE
(on hold until Jan 15) Propose RFC: Forward Protocol

### PR DESCRIPTION
**Pull request by dhh1128**

### CODE REVIEW

**Comment by kdenhartog** [comment]
once you resolve the conflicts on this I'll merge it. Also is there a way that we can make it so the merge conflicts don't occur? They always end up being because of the index.md file.

**Comment by swcurran** [comment]
What about having the maintainers updating the index.md when doing a merge? It is a few extra steps. Not sure if that would break the CI/CD.

**Comment by dhh1128** [comment]
Please do NOT merge this yet, @kyle den hartog <kdenhar@gmail.com> . I want
to update a bit first. I'll ping you again soon.

Maintainers can already fix the index.md, and I often do that to make
things go faster. DM me if you want the recipe.

File: features/0274-forward-protocol/README.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+The only message in this protocol is the `forward` message. A simple and common version of a `forward` message might look like this:
+
+```jsonc
+{
+    "@type": "https://didcomm.org/routing/2.0/forward",
+    "to"   : "did:foo:1234abcd",
+    "payloads~attach": [
+        // One payload
+    }
```
Comment (dbluhm) [inline features/0274-forward-protocol/README.md]: Should be a `]`

**Comment by kdenhartog** [comment]
@dhh1128 did you get time to finish this work up yet? I haven't seen commits since you changed the dates.

**Comment by dhh1128** [comment]
No. I guess I have to reset the dates again. Sorry.

**Comment by kdenhartog** [comment]
No worries. Just want to make sure I don't merge it if you don't feel it's ready.

**Comment by dhh1128** [comment]
Instead of keeping this PR open but on hold, I'll just re-raise it when it's ready. That way we don't have continual dissonance about its status.

**Pull request closed without merging**